azurerm/internal/services/recoveryservices/resource_arm_backup_policy_vm.go

```diff
@@ -313,6 +314,10 @@ func resourceArmBackupProtectionPolicyVMCreateUpdate(d *schema.ResourceData, met
 			return tf.ImportAsExistsError("azurerm_backup_policy_vm", *existing.ID)
 		}
 	}
+	// Less than 7 daily backups is no longer supported for create/update
+	if d.HasChange("retention_daily.0.count") && d.Get("retention_daily.0.count").(int) < 7 {
+		return fmt.Errorf("creating/updating `retention_daily` count of less than 7 is not supported")
```

Comment:
in this instance however we probably want to clarify what's going on here:

```suggestion
		return fmt.Errorf("The Azure API has recently changed behaviour so that provisioning a `count` for the `retention_daily` field can no longer be less than 7 days for new/updates to existing Backup Policies. Please ensure that `count` is less than 7, currently %d", d.Get("retention_daily.0.count").(int))
```